### ISSUE TITLE
No button icons in underground 2

### ISSUE THREAD

**Issue by bisquickpancakemix** [xan1242/NFSU-XtendedInput#26]
There are no button icons in the game with this mod, I turned them off in widescreen fix. The functions work correctly but no button icons. I noticed that unlike this mod for the other games, there's no option to change the controller type in the ini script and there's also no global buttons.tpk file either. By the way xtendedinput for undercover has the same issue, no button icons, missing option to change icons in ini and no global buttons tpk file.

**Comment by xan1242** [comment]
Yes, for Underground 2 and Undercover this hadn't yet been implemented for specific reasons for each game.

It's on the TODO list and it will likely be done as soon as I can get to it.

**Comment by eskoONE** [comment]
what are the reason, if i may ask? if its because you need to create the assets for it, i might be able to help with that.

**Comment by HarGabt** [comment]
@eskoONE Because PC version in terms of showing prompts and PC controls like keyboard and mouse is coded much different, without controller in mind, thus Black Box felt unnecessary to do all this stuff with keyboard prompts unlike they did with UG1 and MW and later. Yes, it's weird.

**Comment by xan1242** [comment]
Sorry, didn't see this in time either.

Yeah the PC version just simply doesn't have the code. Assets aren't the issue, it's just purely code related. I'll reimplement them as soon as bigger issues are resolved (namely the configuration menu)

**Comment by ROGsamurai** [comment]
any updates on the icons showing?

**Comment by ROGsamurai** [comment]
Guess not
